model/task/task_testresult_test.go

```diff
@@ -90,81 +90,7 @@ func (s *TaskTestResultSuite) TestNoOldNoNewTestResults() {
 	s.Equal("project-2", t.Project)
 	s.Equal("revision-2", t.Revision)
 
-	s.Empty(t.TestResults)
-}
-
-func (s *TaskTestResultSuite) TestOldNoNewTestResults() {
```

Comment:
Are you removing these because they break with the current changes, or as a prelude to refactoring?